heuristic/destroy_operators/__init__.py

```diff
@@ -10,6 +10,6 @@
 D_OPERATORS: List[Callable[[Solution, RandomState], Solution]] = [
     nearest_pairs,
     random_customers,
-    random_route,
+    random_route
```

Comment:
Prefer to keep the comma here. It's not really needed, but I am in the habit of doing so since I read something similar to [this](https://stackoverflow.com/a/17492103/4316405).